### PR TITLE
v2.1, v3.0, dev/reference: fix a mistake in Lightning configuration file

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

This PR corrects a mistake in the Lightning configuration file.

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1573

### Which version does your change affect? 

`dev`, `v3.0`, `v2.1`

### Checklist 

- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata

### CODE REVIEW

File: v2.1/reference/tools/tidb-lightning/deployment.md [hunk 1]
```diff
@@ -170,9 +170,9 @@ You can find deployment instructions in [TiDB Quick Start Guide](https://pingcap
 
 Download the TiDB-Lightning package (choose the same version as that of the TiDB cluster):
 
-- **v2.1.9**: https://download.pingcap.org/tidb-v2.1.9-linux-amd64.tar.gz
-- **v2.0.9**: https://download.pingcap.org/tidb-lightning-v2.0.9-linux-amd64.tar.gz
-- Latest unstable version: https://download.pingcap.org/tidb-lightning-test-xx-latest-linux-amd64.tar.gz
+- **v2.1.9**: `https://download.pingcap.org/tidb-v2.1.9-linux-amd64.tar.gz`
```
Comment (lilin90) [inline v2.1/reference/tools/tidb-lightning/deployment.md]: I don't think we need the inline code format. Without it, users can get it directly by clicking the link. @dcalvin What do you think?

Comment (dcalvin) [reply v2.1/reference/tools/tidb-lightning/deployment.md]: Agreed. URLs have their own format already. Inlining them will invalidate the link.

File: dev/reference/tools/tidb-lightning/deployment.md [hunk 2]
```diff
@@ -417,9 +417,13 @@ Download the TiDB-Lightning package (choose the same version as that of the TiDB
     [post-restore]
     # Performs `ADMIN CHECKSUM TABLE <table>` for each table to verify data integrity.
     checksum = true
-    # Performs level-1 compaction after importing each table.
+    # If the value is set to `true`, a level-1 compaction is performed
+    # every time a table is imported.
+    # If unset, it is set to `false` by default.
```
Comment (dcalvin) [inline dev/reference/tools/tidb-lightning/deployment.md]:
```suggestion
    # The default value is `false`.
```

Comment (dcalvin) [reply dev/reference/tools/tidb-lightning/deployment.md]: "if unset" = "by default". So this sentence has a redundant structure.

File: dev/reference/tools/tidb-lightning/deployment.md [hunk 3]
```diff
@@ -417,9 +417,13 @@ Download the TiDB-Lightning package (choose the same version as that of the TiDB
...
     level-1-compact = false
-    # Performs full compaction on the whole TiKV cluster at the end of process.
+    # If the value is set to `true`, a full compaction on the whole
+    # TiKV cluster is performed at the end of the import.
+    # If unset, it is set to `false` by default.
```
Comment (dcalvin) [inline dev/reference/tools/tidb-lightning/deployment.md]: ditto

File: dev/reference/tools/tidb-lightning/deployment.md [hunk 4]
```diff
@@ -170,9 +170,9 @@ You can find deployment instructions in [TiDB Quick Start Guide](https://pingcap
 
 Download the TiDB-Lightning package (choose the same version as that of the TiDB cluster):
```
Comment (ericsyh) [inline dev/reference/tools/tidb-lightning/deployment.md]:
```suggestion
Download the TiDB Lightning package (choose the same version as that of the TiDB cluster):
```

File: v2.1/reference/tools/tidb-lightning/deployment.md [hunk 5]
```diff
@@ -170,9 +170,9 @@ You can find deployment instructions in [TiDB Quick Start Guide](https://pingcap
 
 Download the TiDB-Lightning package (choose the same version as that of the TiDB cluster):
```
Comment (ericsyh) [inline v2.1/reference/tools/tidb-lightning/deployment.md]:
```suggestion
Download the TiDB Lightning package (choose the same version as that of the TiDB cluster):
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 6]
```diff
@@ -2,7 +2,6 @@
 title: TiDB-Lightning Deployment
 summary: Deploy TiDB-Lightning to quickly import large amounts of new data.
 category: reference
-aliases: ['/docs/tools/lightning/deployment/']
 ---
 
 # TiDB-Lightning Deployment
```
Comment (ericsyh) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
# TiDB Lightning Deployment
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 7]
```diff
@@ -171,9 +170,9 @@ You can find deployment instructions in [TiDB Quick Start Guide](https://pingcap
 
 Download the TiDB-Lightning package (choose the same version as that of the TiDB cluster):
```
Comment (ericsyh) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
Download the TiDB Lightning package (choose the same version as that of the TiDB cluster):
```

**Comment by ericsyh** [comment]
LGTM

**Comment by lilin90** [comment]
@TomShawn Please address comments. Thanks!

**Pull request merged**